src/database/models/triprequest.js

```diff
@@ -0,0 +1,40 @@
+const { Model } = require('sequelize');
+
+export default (sequelize, DataTypes) => {
+  class TripRequest extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      TripRequest.belongsTo(
+        models.User,
+
+        {
+          foreignKey: {
+            name: 'requesterId',
+          },
+        }
+      );
+    }
+  }
+  TripRequest.init(
+    {
+      requesterId: DataTypes.INTEGER,
+      managerId: DataTypes.INTEGER,
+      departure_place: DataTypes.STRING,
+      destination: DataTypes.STRING,
+      departureDate: DataTypes.DATEONLY,
+      returnDate: DataTypes.DATEONLY,
+      travel_reason: DataTypes.STRING,
+      accomodationId: DataTypes.INTEGER,
+      status: DataTypes.ENUM('pending', 'approved', 'declined'),
```

Comment:
please change the status: 'declined' to 'rejected'

Reply:
done